src/components/Footer.tsx

```diff
@@ -0,0 +1,80 @@
+// TodoFooter.tsx
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+interface FooterProps {
+  todos: Todo[];
+  todosLeft: number;
+  filter: 'all' | 'active' | 'completed';
+  onFilterChange: (filter: 'all' | 'active' | 'completed') => void;
+  loading: boolean;
+}
+
+const Footer: React.FC<FooterProps> = ({
+  todos,
+  todosLeft,
+  filter,
+  onFilterChange,
+  loading,
+}) => {
+  return (
+    <>
+      {todos.length > 0 && (
+        <footer className="todoapp__footer" data-cy="Footer">
+          <span className="todo-count" data-cy="TodosCounter">
+            {todosLeft} item{todosLeft !== 1 ? 's' : ''} left
+          </span>
+
+          {/* Active link should have the 'selected' class */}
+          <nav className="filter" data-cy="Filter">
+            <a
+              href="#/"
+              className={classNames('filter__link', {
+                selected: filter === 'all',
+              })}
+              data-cy="FilterLinkAll"
+              onClick={() => onFilterChange('all')}
+            >
+              All
+            </a>
+
+            <a
+              href="#/active"
+              className={classNames('filter__link', {
+                selected: filter === 'active',
+              })}
+              data-cy="FilterLinkActive"
+              onClick={() => onFilterChange('active')}
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method